src/components/HeroBanner/HeroBanner.module.scss

```diff
@@ -0,0 +1,44 @@
+.outerBg {
+  height: 420px;
+  position: relative;
+}
+
+.bgWrap {
+  height: 100%;
+  width: 100%;
+}
+
+.logoWrap {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+
+  background: var(--color-neutral-400);
+  border-radius: 50%;
+  height: 250px;
+  width: 250px;
+  padding: 20px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  @media (min-width: 390px) {
+    width: 300px;
+    height: 300px;
+    padding: 26px;
+  }
+
+  @media (min-width: 768px) {
+    width: 350px;
+    height: 350px;
+    padding: 25px;
+  }
+}
+
+.logo {
```

Comment:
Since the logo is an .svg file, you should be able to import it directly by implementing something like SVGR: https://frontend-digest.com/how-to-import-svgs-into-nextjs-8ec6100e613f

Reply:
Importing an SVG directly like that gives you the ability to color it like how you would color text.